FEM/rf_random_walk.cpp

```diff
@@ -6013,26 +6012,177 @@ void DATWriteParticleFile(int current_time_step)
 	// Write particle identities
 	vtk_file << "\n";
 	vtk_file << "POINT_DATA " << RW->numOfParticles << "\n";
-	vtk_file << "SCALARS identity float 1"
-	         << "\n";
-	vtk_file << "LOOKUP_TABLE default"
-	         << "\n";
+	vtk_file << "SCALARS identity int 1" << "\n";
+	vtk_file << "LOOKUP_TABLE default" << "\n";
 	for (int i = 0; i < np; ++i)
 		vtk_file << RW->X[i].Now.identity << "\n";
 
 	// Write particle on_boundary or not
 	vtk_file << endl;
-	vtk_file << "SCALARS on_boundary float 1" << endl;
+    vtk_file << "SCALARS on_boundary int 1" << endl;
 	vtk_file << "LOOKUP_TABLE default" << endl;
 	for (int i = 0; i < np; ++i)
 		vtk_file << RW->X[i].Now.on_boundary << endl;
 
 	// Write particle vectors
 	vtk_file << endl;
-	vtk_file << "VECTORS velocity float" << endl;
+	vtk_file << "VECTORS velocity double" << endl;
 	for (int i = 0; i < np; ++i)
 		vtk_file << RW->X[i].Now.Vx << " " << RW->X[i].Now.Vy << " " << RW->X[i].Now.Vz << endl;
 
 	// Let's close it, now
 	vtk_file.close();
 }
+
+/**************************************************************************
+   OpenGeoSys- Function: DATWriteFile
```

Comment:
Should be `DATWriteParticleControlPlaneFile` but in my opinion the whole comment can be removed.